pipelines/batch/src/main/java/com/google/fhir/analytics/model/BulkExportHttpResponse.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020-2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.fhir.analytics.model;
+
+import java.util.Date;
+import lombok.Data;
+import org.hl7.fhir.instance.model.api.IBaseOperationOutcome;
+
+/** Model class for holding the details of the bulk export job output */
+@Data
```

Comment:
For pure data objects, I have found the "new" [record](https://docs.oracle.com/en/java/javase/17/language/records.html) construct is also useful.

Reply:
Used  [Lombok's @Builder](https://projectlombok.org/features/Builder) annotation.

Reply:
Please make all fields `final` too.

Reply:
Apparently, we can use Lombok's `@Builder` with `record` too, according to [this](https://github.com/projectlombok/lombok/issues/2851). I don't mind not using `record` (as long as we change the fields to `final`); just wanted to clarify the earlier conversation.

Reply:
This is great. It solves the problem of setting the attributes conditionally using Builder pattern. I have made these changes in the latest commit.